src/server/routers/user/_utils/user-breadcrumbs.ts

```diff
@@ -0,0 +1,279 @@
+import { ValidatedSegments } from "@/lib/validations/breadcrumbs";
+import {
+  adminRoutes,
+  instanceTabs,
+  studentRoutes,
+  supervisorRoutes,
+} from "@/lib/validations/instance-tabs";
+import { InstanceParams } from "@/lib/validations/params";
+import { isGroupAdmin } from "@/server/utils/is-group-admin";
+import { isSubGroupAdmin } from "@/server/utils/is-sub-group-admin";
+import { isSuperAdmin } from "@/server/utils/is-super-admin";
+import { PrismaClient, Role } from "@prisma/client";
+
+/**
+ * Depending on the url route params we can infer where the user is (or thinks they are)
+ * @param path
+ */
```

Comment:
Can you expand on what you mean by this? why its relevant? Or just remove the comment.

Reply:
Oh my bad, that should just get removed, I started writing out what I wanted to do but then moved my notes elsewhere.